makefile

```diff
@@ -0,0 +1,83 @@
+.PHONY=all chebi-subsets
+
+# All ------------------------------------------------------------------------------------------------------------------
+all: chebi-subsets
+
+# Analysis -------------------------------------------------------------------------------------------------------------
+input/analysis/:
+	mkdir -p $@
+
+output/analysis/:
+	mkdir -p $@
+
+# - ChEBI subsets
+PART_MAPPINGS=loinc_release/Loinc_2.78/AccessoryFiles/PartFile/PartRelatedCodeMapping.csv
+CHEBI_OWL=input/analysis/chebi.owl
+CHEBI_MODULE=output/analysis/chebi_module.txt
+CHEBI_OUT_BOT=output/analysis/chebi-subset-BOT.owl
+CHEBI_OUT_MIREOT=output/analysis/chebi-subset-MIREOT.owl
+
+chebi-subsets: $(CHEBI_OUT_BOT) $(CHEBI_OUT_MIREOT)
+
+input/analysis/chebi.owl.gz: | input/analysis/
+	wget -O $@ ftp://ftp.ebi.ac.uk/pub/databases/chebi/ontology/chebi.owl.gz
+
+input/analysis/chebi.owl: input/analysis/chebi.owl.gz
+	gunzip -c $< > $@
+	rm $<
+
+# todo: bug fix for label comment: Alwyas shows up as ' # ,'. Alternatively, I could just not include the label comment.
+$(CHEBI_MODULE): $(PART_MAPPINGS) | output/analysis/
+	awk -F'",' '/ebi\.ac\.uk\/chebi/ { \
```

Comment:
## Extracting module via `awk`
@ShahimEssaid For extracting the unique list of mapped ChEBI terms from the LOINC mapping CSV, I wanted to not rely on Python for this, but maybe I'll change that for a few reasons:
1. Windows doesn't come with `awk`, etc.
2. Couldn't get it to display commented labels next to the terms. 
3. The `awk` command, and parsing CSV with default unix commands, is still non-trivial. It's hard to read / maintain.
4. Has an off-by-1 (row) error.

Reply:
If its easy enough, write a sssom adapter and add to sssom-py? Note that you will have other mappings to process, e.g. https://github.com/monarch-initiative/monarch-mapping-commons/pull/35, since the LOINC part mappings is always incomplete, so makes sense to try and convert everything to sssom first